mu/modes/adafruit.py

```diff
@@ -57,6 +57,10 @@ class AdafruitMode(MicroPythonMode):
         (0x2B04, 0xC00C),  # Particle Argon
         (0x2B04, 0xC00D),  # Particle Boron
         (0x2B04, 0xC00E),  # Particle Xenon
+        (0x1209, 0xBAB1),  # Electronic Cats Meow Meow
+        (0x1209, 0xBAB2),  # Electronic Cats CatWAN USBStick
+        (0x1209, 0xBAB3),  # Electronic Cats Bast Pro Mini M0
+        (0x1209, None),       # Electronic Cats future Board
```

Comment:
I believe any open source project can register a PID with the 0x1209 VID, so we shouldn't reserve the entire range for this.

Reply:
ok, then delete and leave only 3 Boards?

Reply:
Yep :)

Reply:
Ready @carlosperate !!!